airdrop/config.py

```diff
@@ -24,8 +24,6 @@
 SIGNER_PRIVATE_KEY = 'AIRDROP_SIGNER_PRIVATE_KEY'
 SIGNER_PRIVATE_KEY_STORAGE_REGION = ''
 
-NUNET_TOKEN_ADDRESS = ''
-STAKING_CONTRACT_ADDRESS = ''
 STAKING_TOKEN_NAME = 'AGIX'
```

Comment:
STAKING_TOKEN_NAME
Should come from the database.

Reply:
Moved to db in the latest commit